### ISSUE TITLE
Use input + range combo control in column block width

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#56520]
I expect the input + range control, as seen `HeightControl` to be used within the column block, instead of what we have in place today — where you are expected to manually enter the value each time. 

It could be interesting to explore 10% increment steps in the RangeControl, if set to percentages (which I think should be the default unit for the width control anyhow. 

### Visuals
![CleanShot 2023-11-24 at 14 53 54](https://github.com/WordPress/gutenberg/assets/1813435/0b8976cf-07f3-4235-b1d5-ba939b985aca)

![CleanShot 2023-11-24 at 14 54 16](https://github.com/WordPress/gutenberg/assets/1813435/174f5710-b719-4168-84bd-c3f6a1636c4a)

**Comment by gravnetic** [comment]
I agree with this preference and would expand on this. I would like to have a preference in settings/styles where I can set my default preferred values for section, div, and even paragraphs: %, vw, calc, em, rem, and then these values are used and made available as described here. This is available in every visual editor, from Photoshop, Illustrator, and Figma to many website solutions.

**Comment by bacoords** [comment]
Love this and the idea that we could opt into loading with `%` by default, instead of pixels.

I would just point out that 10% increments feel pretty arbitrary. What are the most common column patterns? I'd bet 50/50, 66/33, 33/33/33, 25/75, or percentages based on a 12-col grid. Very few of those real-world situations would apply to a 10% step.

**Comment by cbirdsong** [comment]
Related: 
- https://github.com/WordPress/gutenberg/issues/55259
